src/it/resources/org/checkstyle/suppressionxpathfilter/innertypelast/SuppressionXpathRegressionInnerTypeLastThree.java

```diff
@@ -0,0 +1,13 @@
+package org.checkstyle.suppressionxpathfilter.innertypelast;
+
+public class SuppressionXpathRegressionInnerTypeLastThree {
+
+    static {} // OK
+
+    interface Inner {
+    }
+
+    public SuppressionXpathRegressionInnerTypeLastThree() { // warn
```

Comment:
Two and Three are almost identical.

I meant:
```
public class SuppressionXpathRegressionInnerTypeLastTwo {

    static {}; // OK
    
    public void innerMethod() {} 

class Inner {



};

}
```

Reply:
done.